src/renderer/components/hotbar/hotbar-menu.tsx

```diff
@@ -52,7 +52,7 @@ export class HotbarMenu extends React.Component<Props> {
       return null;
     }
 
-    return item ? catalogEntityRegistry.items.find((entity) => entity.metadata.uid === item.entity.uid) : null;
+    return item ? catalogEntityRegistry.items.find((entity) => entity?.metadata.uid === item.entity.uid) : null;
```

Comment:
Is this still needed then?

Reply:
no, removed https://github.com/lensapp/lens/pull/2814/commits/79740943be47f4414e715b1876c1992591184b94